drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -117,6 +117,11 @@ matterGeneric:
     deviceTypes:
       - id: 0x002C
     deviceProfileName: air-quality-sensor
+  - id: "matter/air-quality-ikea"
+    deviceLabel: IKEA air Quality Sensor
```

Comment:
```suggestion
    deviceLabel: IKEA Air Quality Sensor
```

Reply:
This fingerprint was removed as recommended in another comment.